### ISSUE TITLE
No Controls Working

### ISSUE THREAD

**Issue by dave2503** [funtastix/rinnaitouch#87]
Hi there i followed everything as it was but none of the controls are working for me.

![image](https://github.com/user-attachments/assets/b00d491e-c4e5-4699-b07c-4f8907afd65f)

**Comment by funtastix** [comment]
Ji,

This indicates that you didn't achieve a connection. Can you check your IP and also ensure the app is not connected? I suggest to kill the app, restart the wifi, check the IP, and try again. If that doesn't work, please follow the instructions to send some debug logs.

**Comment by dave2503** [comment]
The ip is definitely correct because when I put that ip in browser i am abke to access the wifi kit settings through the ip. It might be the phone thing you mentioned but how do I remove my phone or kill the app as u suggested

**Comment by chanpreetsal** [comment]
i am not not able to get it to wokr. I have spent a lot of time trying different things for it to connect. The integration works fine on Home Bridge via mantorok1. 

When i heck the debug log i see lots of errors around like these.

2024-11-27 10:22:22.078 ERROR (Thread-23 (receiver)) [pyrinnaitouch.receiver] Socket timed out, renewing connection (TimeoutError('timed out'))
2024-11-27 10:22:24.432 ERROR (Thread-21 (receiver)) [pyrinnaitouch.receiver] Socket timed out, renewing connection (OSError(9, 'Bad file descriptor'))

I wanted to confirm, if the integration is still active and does it work for others?

**Comment by funtastix** [comment]
Are you running both integrations at the same time? The unit can only connect to one local device at once

**Comment by chanpreetsal** [comment]
> Are you running both integrations at the same time? The unit can only connect to one local device at once

No. I have two wifi controllers at home. One is connected to home assistant and the other to Home bridge.   Limitation of Home Bridge is the same that i can only connect to one controller or else i have to run it as a child bridge.

Homebridge has a nice logging capability that shows me that its connected and working fine and if i enter the wifi key, it automatically restarts every midnight. I feel this recycling keeps the wifi controller working fine.  Its easy to identify the  status. 
![Rinnah temp](https://github.com/user-attachments/assets/3d8b0dec-0a45-4a95-970c-03247b4d1943)

I have tried numerous tries to move to HA plugin for this, as i can use the automations then but i am not able to reliability connect or stay connected.

**Comment by KellsonM** [comment]
> i am not not able to get it to wokr. I have spent a lot of time trying different things for it to connect. The integration works fine on Home Bridge via mantorok1.
> 
> When i heck the debug log i see lots of errors around like these.
> 
> 2024-11-27 10:22:22.078 ERROR (Thread-23 (receiver)) [pyrinnaitouch.receiver] Socket timed out, renewing connection (TimeoutError('timed out')) 2024-11-27 10:22:24.432 ERROR (Thread-21 (receiver)) [pyrinnaitouch.receiver] Socket timed out, renewing connection (OSError(9, 'Bad file descriptor'))
> 
> I wanted to confirm, if the integration is still active and does it work for others?

I got similar issue, checked IP, port etc. HA keep getting the above log and filling up the storage :( 

Another interesting thing is the Rinnai WiFi modual web page access seems stop working after a while after every restart, but this doesn't impact on the Rinnai Touch Mobile app though.

**Comment by Wirepower** [comment]
I’m having the exact same issue. Would love an update on this

**Comment by signalbug** [comment]
I am also having the same issue.
